src/components/Property/Types/NumericProperty/NumericProperty.tsx

```diff
@@ -0,0 +1,84 @@
+import { Flex, Group } from '@mantine/core';
+
+import { usePropListeningState } from '@/api/hooks';
+import { NumericInput } from '@/components/Input/NumericInput/NumericInput';
+import { PropertyLabel } from '@/components/Property/PropertyLabel';
+
+import { NumericPropertySlider } from './NumericPropertySlider';
+
+interface Props {
+  name: string;
+  description: string;
+  disabled: boolean;
+  setPropertyValue: (newValue: number) => void;
+  value: number;
+  additionalData: {
+    Exponent: number;
+    MaximumValue: number;
+    MinimumValue: number;
+    SteppingValue: number;
+  };
+  // TODO: view options in metadata
+}
+
+export function NumericProperty({
+  name,
+  description,
+  disabled,
+  setPropertyValue,
+  value,
+  additionalData
+}: Props) {
+  const { value: currentValue, set: setCurrentValue } =
+    usePropListeningState<number>(value);
+
+  const min = additionalData.MinimumValue;
+  const max = additionalData.MaximumValue;
+  const step = additionalData.SteppingValue;
+  const exponent = additionalData.Exponent;
+
+  const isInt = step >= 1;
+
+  // When no min/max is set, the marks for the slider cannot be nicely computed
+  const hasNiceMinMax = isFinite(max - min);
+
+  // @TODO There still seems to be a bit of a stutter when dragging the slider.
+  // nvestigate. Is there somethign we could memo to make it better?
+  function onValueChange(newValue: number) {
```

Comment:
onValueEnd? or something (update the value when releasing the slider)

Reply:
Unfortunately, I donät think that is an option.  The user probably want to see the value bein updated while the slider is being used